{"name": "test_login_usernameFormat_error", "status": "passed", "attachments": [{"name": "log", "source": "8cde2d7c-e858-4f07-a3cb-5d8ab743c047-attachment.txt", "type": "text/plain"}], "start": 1610612434973, "stop": 1610612438799, "uuid": "52d600ed-fc99-4395-8cce-9da54e53bee9", "historyId": "b7c9963723357b0faf6d03130d678f90", "testCaseId": "e5bc1aeb101de12749be698a7235f84d", "fullName": "TestCases.test_logincase.test_login.Test_Login#test_login_usernameFormat_error", "labels": [{"name": "story", "value": "测试账号错误"}, {"name": "parentSuite", "value": "TestCases.test_logincase"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "Test_Login"}, {"name": "host", "value": "USER-20190821YP"}, {"name": "thread", "value": "6000-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "TestCases.test_logincase.test_login"}]}